http/src/test/java/com/github/awsjavakit/http/OAuth2HttpClientTest.java

```diff
@@ -108,10 +107,11 @@ void shouldReturnTheSameTagAsTheTokenProvider() {
 
   @Test
   void shouldRetrySyncRequestsBasedOnTheRetryPolicy() throws IOException, InterruptedException {
```

Comment:
Is this test here needed if the OAuth2HttpClient is back to normal not retrying?